qdrant-landing/content/articles/vector-similarity.md

```diff
@@ -0,0 +1,211 @@
+
+Qdrant also supports vector [quantization](https://qdrant.tech/documentation/guides/quantization/) to reduce memory footprint and speed up vector similarity searches, making it very effective for large-scale applications where efficient resource management is critical.
+
+There are three quantization strategies you can choose from - scalar quantization, binary quantization and product quantization - which will help you control the trade-off between storage efficiency, search accuracy and speed.
+
+**Security**
+
+Qdrant offers several [security features](https://qdrant.tech/documentation/guides/security/) to help protect data and access to the vector store:
+
+- API Key Authentication: This helps secure API access to Qdrant Cloud with static or read-only API keys.
+- JWT-Based Access Control: You can also enable more granular access control through JSON Web Tokens (JWT), and opt for restricted access to specific parts of the stored data while building Role-Based Access Control (RBAC).
+- TLS Encryption: Additionally, you can enable TLS Encryption on data transmission to ensure security of data in transit.
+
+To help with data sovereignty, Qdrant can be run in a [Hybrid Cloud](https://hybrid-cloud.qdrant.tech/) setup. Hybrid Cloud allows for seamless deployment and management of the vector database across various environments, and integrates Kubernetes clusters into a unified managed service. You can manage these clusters via Qdrant Cloud’s UI while maintaining control over your infrastructure and resources.
+
+## **Optimizing Similarity Search Performance**
+
+In order to achieve top performance in vector similarity searches, Qdrant employs a number of other tactics in addition to the features discussed above.
+
+**FastEmbed**: Qdrant supports [FastEmbed](https://qdrant.tech/articles/fastembed/), a lightweight Python library for generating fast and efficient text embeddings. FastEmbed uses quantized transformer models integrated with ONNX Runtime, and is significantly faster than traditional methods of embedding generation.
+
+**Support for Dense and Sparse Vectors**: Qdrant supports both dense and sparse vector representations. While dense vectors are most common, you may encounter situations where the dataset contains a range of specialized domain-specific keywords. [Sparse vectors](https://qdrant.tech/articles/sparse-vectors/) shine in such scenarios. Sparse vectors are vector representations of data where most elements are zero.
+
+**Multitenancy**: Qdrant supports [multitenancy](https://qdrant.tech/documentation/guides/multiple-partitions/) by allowing vectors to be partitioned by payload within a single collection. Using this you can isolate each user's data, and avoid creating separate collections for each user. In order to ensure indexing performance, Qdrant also offers ways to bypass the construction of a global vector index, so that you can index vectors for each user independently.
```

Comment:
```suggestion
**Multitenancy**: Qdrant supports [multitenancy](/documentation/guides/multiple-partitions/) by allowing vectors to be partitioned by payload within a single collection. Using this you can isolate each user's data, and avoid creating separate collections for each user. In order to ensure indexing performance, Qdrant also offers ways to bypass the construction of a global vector index, so that you can index vectors for each user independently.
```